deafrica/data/gmw.py

```diff
@@ -138,28 +138,26 @@ def gmw_download_stac_cog(year: str, s3_dst: str) -> None:
         )
         check_output(cmd, stderr=STDOUT, shell=True)
 
+        io_timer(file_path=output_file, log=log)
```

Comment:
WHy not just log the time?

Reply:
This function isn't logging the time, it's giving Argo time to write the file before go ahead.